perun-core/src/main/java/cz/metacentrum/perun/core/bl/TasksManagerBl.java

```diff
@@ -162,6 +164,8 @@
 
 	void removeTask(Service service, Facility facility);
 
+	void removeAllTasksForService(Service service);
```

Comment:
Please, add also a javadoc for this method.

Reply:
Added.